src/track/text-style.js

```diff
@@ -257,9 +257,20 @@ class TextStyle {
     attributes.push('color: ' + TextStyle.toRGBA(this.fontColor, this.fontOpacity));
     attributes.push('background-color: ' + TextStyle.toRGBA(this.backgroundColor, this.backgroundOpacity));
     attributes.push('text-shadow: ' + this.getTextShadow());
+    attributes.push('font-size: ' + this.fontSize);
     return attributes.join('!important; ');
   }
 
+  toCSSObject(): string {
```

Comment:
where is this used ?

Reply:
so the bug was with the font size only ?

Reply:
Right